src/index.js

```diff
@@ -27,9 +27,7 @@ export default class Swup {
 		let defaults = {
 			animateHistoryBrowsing: false,
 			animationSelector: '[class*="transition-"]',
-			linkSelector: `a[href^="${
-				window.location.origin
-			}"]:not([data-no-swup]), a[href^="/"]:not([data-no-swup]), a[href^="#"]:not([data-no-swup])`,
+			linkSelector: `a[href^="${window.location.origin}"], a[href^="/"], a[href^="#"]`,
```

Comment:
This could be a good chance to also add the selector `a[href^="./"]`. That kind of link is necessary in combination with the [`<base>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) element.